### ISSUE TITLE
Error when i saved my facet

### ISSUE THREAD

**Issue by Loubal70** [AmphiBee/wpgb-extended#24]
Hi!
Since the last update (v2.0.5), with the new panel, I encounter an error when I save my facet:
"Class 'WP_Grid_Builder\Admin\Export' not found"

This error does not block the save operation.
The problem is located here: htdocs/content/mu-plugins/wpgb-extended/src/Providers/ItemSync.php

**Comment by ogorzalka** [comment]
Yes since the last major update, the export process has been reworked. It's a known issue but I have no idea when I will be able to fix it 😀
